github_is_so_cool/ReadMe.md

```diff
@@ -0,0 +1,3 @@
+[Github](https://www.github.com) is one of the coolest thing a software developer could think of.
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.